### PR TITLE
CHANGELOG: add release notes for v3.0.2

### PR DESCRIPTION
**Pull request by bb7133**
### What problem does this PR solve? 
Add release note for v3.0.2

### CODE REVIEW

**Comment by bb7133** [comment]
PTAL @winkyao @jackysp @zz-jason

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,95 @@
 # TiDB Changelog
 All notable changes to this project will be documented in this file. See also [Release Notes](https://github.com/pingcap/docs/blob/master/releases/rn.md), [TiKV Changelog](https://github.com/tikv/tikv/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+
+## [3.0.2] 2019-08-06
+### SQL Optimizer
+* Fix the issue that the “Can’t find column in schema” message is reported when the same table occurs multiple times in a query and logically the query result is always empty [#11247](https://github.com/pingcap/tidb/pull/11247)
```
Comment (winkyao) [inline CHANGELOG.md]: `“` --> `"`, `’` --> "'"

**Comment by winkyao** [comment]
@bb7133 Please address comments

**Comment by bb7133** [comment]
@winkyao Done, thanks

**Pull request merged**